tools/certa/src/runners/electron/ElectronTestRunner.ts

```diff
@@ -35,7 +36,7 @@ export class ElectronTestRunner {
         contextIsolation: false,
         nativeWindowOpen: true,
         sandbox: false,
-      },
+      } as WebPreferences, // nativeWindowOpen was removed starting Electron 18
```

Comment:
Why not delete the `nativeWindowOpen: true,` line instead of casting?

Reply:
In Electron 14 (which Certa still supports) `nativeWindowOpen` would default to `false`. Might break something for tests still using Electron 14.

Reply:
Consider using a type like `WebPreferences & { nativeWindowOpen: true }` rather than completely bypassing the type checker with `as`.